code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm

```diff
@@ -15,9 +15,10 @@
 		/datum/design/autolathe/tool/tacknife,
 		/datum/design/autolathe/tool/combat_shovel,
 		/datum/design/autolathe/sec/beartrap,
-		/datum/design/autolathe/device/landmine = 2, //can make 10 warcrimes
+		//datum/design/autolathe/device/landmine = 2, //can make 10 warcrimes //why did you put this here this was an awful idea.
```

Comment:
```suggestion
		/datum/design/autolathe/device/landmine = 2, //can make 10 warcrimes
```

Reply:
please god get rid of the landmines on the misc disk it makes no sense to make explosives so easily accessible by everyone

Reply:
Dont remember landmines